microsoft-edge/toc.yml

```diff
@@ -1465,7 +1465,10 @@
         - name: Operating System Regional Data Display
           href: web-platform/os-regional-settings.md
           displayName: globalization, language, region, limited, intl, date, time, format
-          
+
+        - name: View formatted JSON
+          href: devtools-guide-chromium/json-viewer/json-viewer.md
+          displayName: JSON viewer, tool
```

Comment:
fyi re: index entry "tool".   That might give good results in this case.  Try typing "tool" into "Find in files" field in upper left - it gives many hits.  

I had to remove comparable index entries from each What's New file b/c too many low-value hits obscured other topics that weren't What's New.

Reply:
Good point. "tool" gives too many hits. Let's just keep "JSON viewer". I expect people will mostly try to type "json" if they're looking for this feature.